source/fileInfo.go

```diff
@@ -177,11 +189,15 @@ func (fi *FileInfo) init() {
 		if (!fi.isLeafBundle && len(parts) == 1) || len(parts) > 1 {
 			section = parts[0]
 		}
-
 		fi.section = section
 
-		fi.uniqueID = helpers.MD5String(filepath.ToSlash(fi.relPath))
+		var contentBaseName = fi.translationBaseName
+		if fi.isLeafBundle && len(parts) > 0 {
+			contentBaseName = parts[len(parts)-1]
+		}
+		fi.contentBaseName = contentBaseName
 
+		fi.uniqueID = helpers.MD5String(filepath.ToSlash(fi.relPath))
```

Comment:
Unrelated change?

Reply:
Seems to be just a bad diff -- this line was right below my insert. I haven't been able to find out why git is marking this as delete/add.